tensorrt_llm/runtime/kv_cache_manager_v2/_core/_kv_cache_manager.py

```
@@ -292,13 +294,31 @@ def __init__(
         self._stats_excluded_kv_cache_ids = set()
 
     def __del__(self) -> None:
-        self.shutdown()
+        try:
+            self.shutdown()
+        except LogicError as e:
+            warnings.warn(str(e))
```

Ruff (0.16.0)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)